packages/eventcatalog/hooks/EventCatalog.tsx

```diff
@@ -26,7 +26,13 @@ export const useUser = () => {
 
 export const useUrl = () => {
   const config = useConfig();
-  const getEditUrl = (url: string) => path.join(config.editUrl, url);
+  const getEditUrl = (url: string) => {
+    const editBaseUrl = new URL(config.editUrl);
+
+    editBaseUrl.pathname = path.join(editBaseUrl.pathname, url);
```

Comment:
Ok to use `path.join` just on the pathname portion, but we need to isolate that by using the `URL` class first so it doesn't clobber anything else about the base URL.